pkg/store/clear.go

```diff
@@ -2,14 +2,35 @@ package store
 
 import (
 	"context"
+	"errors"
+	"fmt"
 	"github.com/nrc-no/core/pkg/logging"
 	"github.com/nrc-no/core/pkg/sql/convert"
 	"go.uber.org/zap"
 	"gorm.io/gorm"
+	"os"
 )
 
+const clearProtectionEnvVar = "VERY_DANGEROUSLY_ALLOW_TO_CLEAR_DB"
```

Comment:
could we define these in a centralized place?

Reply:
We could, but this one is really a one-off, and the user will get this message when doing `go run . clear` .. 
Also, it's not clear where we would put them, since we don't use env vars anyway, (we use cobra/viper `yaml` config file)